ovos_number_parser/numbers_pt.py

```
@@ -1,503 +1,533 @@
+
+    # --- Integer Pronunciation Logic ---
+    n = int(number)
+
+    # Base case for recursion: numbers less than 1000
+    if n < 1000:
+        return _pronounce_up_to_999(n, variant)
+
+    scale_definitions = _SCALES[scale][variant]
+
+    # Find the largest scale that fits the number
+    for scale_val, s_name, p_name in scale_definitions:
```

Ruff (0.12.2)

Loop control variable `s_name` not used within loop body

(B007)